### PR TITLE
Update container

### PR DESCRIPTION
**Pull request by charlotte-avery**
PR required to address deployment error:`Error: The v0 series of google-github-actions/get-gke-credentials is no longer maintained. It will not receive updates, improvements, or security patches.`

### CODE REVIEW

File: .github/workflows/container.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: google-github-actions/setup-gcloud@v0.2.0
+      - uses: google-github-actions/setup-gcloud@v2
```
Comment (shengy90) [inline .github/workflows/container.yaml:10]: I believe you need to auth first before you can setup gcloud with V2:
https://github.com/google-github-actions/setup-gcloud?tab=readme-ov-file#usage

We had this error on Friday where without authenticating, setup-gcloud fails (though GH action allowed you to continue unauthenticated).

**Review by shengy90 — CHANGES_REQUESTED** [review]
Need to auth first before you can setup gcloud I think with V2

File: .github/workflows/container.yaml (hunk 1 above)
Comment (charlotte-avery) [reply .github/workflows/container.yaml]: Reverted back to v0.2.0

**Review by shengy90 — APPROVED** [review]
LGTM

**Pull request merged**